src/navigation/drive.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+from geometry_msgs.msg import Twist
+from typing import Tuple
+
+#
+def get_drive_command(target_pos : np.ndarray, rover_pos : np.ndarray, rover_dir : np.ndarray, completion_tolerance : float, DRIVE_FWD_THRESH : float) -> Tuple[Twist, bool]:
```

Comment:
We should probably rename DRIVE_FWD_THRESH and make it lowercase now that it is a parameter. Something that includes the fact it is an angular alignment. I like the pydoc, much cleaner to read.